ctapipe/io/datalevels.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum, auto
+
+
+class DataLevel(Enum):
+    '''Enum of the different Data Levels'''
+    R0 = auto()
+    R1 = auto()
+    R2 = auto()
+    DL0 = auto()
+    DL1a = auto()
+    DL1b = auto()
```

Comment:
we don't really have DL1a/b anymore. I'd just drop them for now (we realized it really depends on the use case, like training, etc, which parameters or images are filled in).  Just call it all DL1 for now until that is settled.

Reply:
But for an event source it is a very big difference if it provides images or parameters. That was the idea behind DLa / DLb here.

Should I call it `DL1_images` ?